elements/pfe-card/src/pfe-card.js

```diff
@@ -59,32 +95,15 @@ class PfeCard extends PFElement {
     this._observer.disconnect();
   }
 
-  attributeChangedCallback(attr, oldValue, newValue) {
-    super.attributeChangedCallback(attr, oldValue, newValue);
-    // Strip the prefix from the attribute
-    attr = attr.replace("pfe-", "");
-    // If the observer is defined in the attribute properties
-    if (this[attr] && this[attr].observer) {
-      // Get the observer function
-      let observer = this[this[attr].observer].bind(this);
-      // If it's a function, allow it to run
-      if (typeof observer === "function") observer(attr, oldValue, newValue);
-    }
-  }
-
-  _basicAttributeChanged(attr, oldValue, newValue) {
-    this[attr].value = newValue;
-  }
-
   // Update the color attribute and contexts
-  _colorChanged(attr, oldValue, newValue) {
-    this[attr].value = newValue;
+  _colorChanged(oldValue, newValue) {
     // Trigger an update in nested components
+    console.log("color changed");
```

Comment:
Can remove console when debugging is done

Reply:
Done.